src/protocols/WebSocket.ts

```diff
@@ -137,6 +171,15 @@ export default class WebSocketProtocol extends BaseProtocolRealtime {
       this.client.onmessage = payload => {
         const data = JSON.parse(payload.data || payload);
 
+        /**
+         * Since Kuzzle 2.10.0
+         * Corresponds to a custom pong response message
+         */
+        if (data && data.p && data.p === 2) {
```

Comment:
you need to make sure that this is a PONG payload, by checking that there are no other properties than the `p` one (imagine a custom API route returning a response containing a numeral "p" property)